crates/proof-impl/evm-ee-stf/src/processor.rs

```diff
@@ -33,7 +33,7 @@ use revm::{
     primitives::{SpecId, TransactTo, TxEnv},
     Database, DatabaseCommit, Evm,
 };
-use strata_reth_evm::set_evm_handles;
+use strata_reth_evm::{constants::BRIDGEOUT_ADDRESS, set_evm_handles};
```

Comment:
Is this really the normal way to do this as opposed to pulling it from the network params file?